src/components/our_projects.rs

```diff
@@ -0,0 +1,167 @@
+use yew::{html, Component, ComponentLink, Html, ShouldRender};
```

Comment:
I suggest sorting imports alphabetically and also by its kind.

I usually do:

```rust
// Introduce local modules to context
mod some_module;

// Import dependency crates (external)
use anyhow::{bail, Error};
use yew;

// Import crate (but not sibling) (think of this as crate-absolute path, similar to Python's)
use crate::components::MyBeautifulComponent;

// Module relative paths
use self::some_sibling_module::Sibling;
```

Reply:
Oh nice, it's a good suggestion, I'll change the code!